cms/djangoapps/contentstore/management/commands/reindex_course.py

```diff
@@ -97,14 +98,21 @@ def handle(self, *args, **options):  # pylint: disable=too-many-statements
                         logging.exception('Search Engine error - %s', exc)
                         return
 
-                    index_exists = searcher._es.indices.exists(index=index_name)  # pylint: disable=protected-access
+                    # Meilisearch engine
```

Comment:
Can we avoid leaking the meilisearch abstraction here? For instance, we could call the `create_indexes` function during tutor init. 

The elasticsearch abstraction has already leaked, so I think we could just write: `if hasattr(searcher, "_es"): ...`

Reply:
@regisb 

> Can we avoid leaking the meilisearch abstraction here? For instance, we could call the create_indexes function during tutor init.

Yep, we can do it that way.. can you add this to https://github.com/overhangio/tutor/pull/1141 ?

> The elasticsearch abstraction has already leaked, so I think we could just write: if hasattr(searcher, "_es"): ...

Yep, I'll do that and catch the MeilisearchError thrown during `--setup` , but nothing else.

Reply:
> Yep, we can do it that way.. can you add this to https://github.com/overhangio/tutor/pull/1141 ?

Done.

Reply:
Perfect, thank you!